lib/core/remove-file.ts

```diff
@@ -0,0 +1,70 @@
+import fs from "fs";
+import path from "path";
+import glob from "glob";
+
+import { alerts } from "./alerts";
+import {
+  getScssPathFromTypeDefinition,
+  getTypeDefinitionPath
+} from "../typescript";
+
+/**
+ * Given a single file remove the generated types if they exist
+ *
+ * @param file the SCSS file to generate types for
+ *
+ */
+
```

Comment:
The majority of the codebase doesn't contain a newline between the JSDoc & function. Can we remove this line?